rgb.go

```diff
@@ -0,0 +1,267 @@
+package keyboard
```

Comment:
This change will likely cause the CI to pass.
`sago35/tinygo-keyboard` needs to successfully execute `go build .`.


```suggestion
//go:build tinygo

package keyboard
```